packages/atlas/src/components/_navigation/SidenavViewer/FollowedChannels.styles.ts

```diff
@@ -13,35 +13,60 @@ export const FollowedChannelsWrapper = styled.div`
 `
 
 export const ChannelsTitle = styled(Text)`
+  font: ${cVar('typographyDesktopH100')};
```

Comment:
Those font attributes are not necessary here, also this variant is not correct, it should use `typographyDesktopH300`. You can achive it using props. `<ChannelsTitle variant="h300" secondary>`

Reply:
All right, I changed it.
![image](https://user-images.githubusercontent.com/96242244/152514753-13522ebf-e65d-44e2-8588-880fe6745835.png)

Reply:
@lcoenengithub I'm really, really sorry but I misled you. I thought it's channel name, not channel section title. Title was correct but channel name has wrong variant.
![Zrzut ekranu 2022-02-4 o 12 04 11](https://user-images.githubusercontent.com/13706246/152518723-843615d4-26d8-4642-8e12-ac812d0aeacc.png)

Reply:
Oh that's all right :) I'm changing it.